### ISSUE TITLE
Package.swift copy Privacy Manifest Resource file

### ISSUE THREAD

**Issue by FONickReichard** [apptentive/apptentive-kit-ios#53]
Environment Details: 
**Version:** 6.5.0
**Dependency Management:** SPM
**Relates To:** Privacy Manifest (#529) commit:  3daaa1002b2291d992d0f05b19ac1722983690fc

Hello, 

Our team has updated to v6.5.0 via SPM. Based on the following path structure: [Sources](https://github.com/apptentive/apptentive-kit-ios/tree/main/Sources)/[ApptentiveKit](https://github.com/apptentive/apptentive-kit-ios/tree/main/Sources/ApptentiveKit)/[Resources](https://github.com/apptentive/apptentive-kit-ios/tree/main/Sources/ApptentiveKit/Resources)/PrivacyInfo.xcprivacy 
I am wondering if the PrivacyInfo.xcprivacy file needs to be copied in the Package.swift file. 

Ex:
```
   targets: [
        .target(
            name: "ApptentiveKit",
            dependencies: [],
            exclude: ["Info.plist"],
            resources: [.copy("Resources/SwiftPM.txt"), .copy("Resources/Distribution.plist"), .copy("Resources/PrivacyInfo.xcprivacy")]
        ),
  ```

Correct me if I'm wrong, I believe this should be included in the clients app's bundle so that the privacy report can be generated after archiving a build. 

Any feedback would be great. 

Thank you

**Comment by RamyaSmg34** [comment]
Hello,

Even I am able to reproduce this issue, without   `.copy("Resources/PrivacyInfo.xcprivacy")` under targets  inside Package.swift file, we are not able to generate Privacy report after archiving a build.

Note: Tried by editing Package.swift file locally as below by including `.copy("Resources/PrivacyInfo.xcprivacy")` and was able to generate Privacy report.
```
 targets: [
        .target(
            name: "ApptentiveKit",
            dependencies: [],
            exclude: ["Info.plist"],
            resources: [.copy("Resources/SwiftPM.txt"), .copy("Resources/Distribution.plist"), .copy("Resources/PrivacyInfo.xcprivacy")]
        ),
```

Can you please look into this and help here

Thank you

**Comment by FONickReichard** [comment]
@frankus Could we get an update on this when you're free. Thanks!

**Comment by frankus** [comment]
@FONickReichard I'll get this prioritized. We seem to have a bit of a gap in our issue ingestion system where this has gotten lost in the shuffle.

**Comment by FONickReichard** [comment]
Thank you for the update @frankus

**Comment by FONickReichard** [comment]
Hey @frankus , 

Friendly check in. Is there an eta for a new version? 

Thanks.

**Comment by frankus** [comment]
Just released as part of the v6.6.0. 

(We're aware of an issue with displaying surveys in this release due to an API rollout taking longer than the SDK release).

**Comment by MichaelWellsSM** [comment]
I believe the same would apply to the podspec... thank you.

**Comment by einsteinx2** [comment]
> I believe the same would apply to the podspec... thank you.

That's correct, it's also missing from the podspec, it must be specified as a resource. Specifically it's missing from this line:

`spec.resource_bundles = { "ApptentiveKitResources" => [ "Sources/ApptentiveKit/Resources/*.lproj", "Sources/ApptentiveKit/Resources/Media.xcassets", "Sources/ApptentiveKit/Resources/Distribution.plist" ] }`

which should be

`spec.resource_bundles = { "ApptentiveKitResources" => [ "Sources/ApptentiveKit/Resources/*.lproj", "Sources/ApptentiveKit/Resources/Media.xcassets", "Sources/ApptentiveKit/Resources/Distribution.plist", "Sources/ApptentiveKit/Resources/PrivacyInfo.xcprivacy" ] }`
